HLTrigger/HLTfilters/plugins/plugins.cc

```diff
@@ -62,6 +70,12 @@ typedef HLTSinglet<l1extra::L1JetParticle>
     HLTLevel1Jet;  // the actual type is ovrridden object-by-object (TriggerL1CenJet, TriggerL1ForJet or TriggerL1TauJet)
 typedef HLTSinglet<l1extra::L1MuonParticle> HLTLevel1Muon;
 
+// filters for Phase-2
+typedef L1JetFilterT<reco::CaloJet> L1JetFilter;
```

Comment:
typedef L1JetFilterT<reco::CaloJet> L1CaloJetFilter;

In general, why are L1T object filers templated on reco objects?

Reply:
Because the L1T implementation produces reco objects. If we look at one event content dump of the HLT TDR samples we will find:
```
vector<reco::PFMET>                   "l1PFMetCalo"               ""                "RECO"
vector<reco::PFMET>                   "l1PFMetPF"                 ""                "RECO"
vector<reco::PFMET>                   "l1PFMetPuppi"              ""                "RECO"
```
and for the sliding window jets, they are a vector of `reco::CaloJet`:
https://github.com/cms-l1t-offline/cmssw/blob/phase2-l1t-integration-CMSSW_11_1_3/L1Trigger/L1CaloTrigger/plugins/Phase1L1TJetProducer.cc#L116